main-cookbooks/teracy-dev/recipes/normalization.rb

```diff
@@ -38,3 +38,14 @@
     EOH
   not_if { ::File.exist?('/etc/bash_completion') && ::File.exist?('/usr/share/bash-completion/bash_completion') }
 end
+
+bash 'set default route external interface' do
```

Comment:
it's better to move this into another PR for the right issue.

Reply:
It's is depend, if the VM using host's network connection for certificate download then .... request to localdev.tk will point to VM IP instead of right ip.

Reply:
this change will not resolve the problem when I `$ vagrant reload`